### PR TITLE
Migrate Stripe related services to Angular

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Migrate Stripe related services to Angular.

Refactored stripeLoader to use Promise and improved the logic for using prod vs test api keys.
Also added some suggestions for future improvements [here](https://docs.google.com/document/d/1Lha7Uf7kGjdhOb95k3V1jyW0xk3L6jYzvD-6PUf3MWg/edit#bookmark=id.nqtzwlg52v0u).

## Motivation and Context
Angular Migration

## How Has This Been Tested?
Locally and on [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/environments/environment.beta.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const environment = {
-  production: false,
+  production: true,
```
Comment (ezequielc) [inline src/environments/environment.beta.ts:2]: @alex-deaconu I want your feedback on this one. I set it to true so we can use the same stripe keys as in production. But I am not sure this is clear enough given the name is `production` and could be misleading in the future. I am considering setting an `env_name` variable and use that instead. Thoughts? Thanks

Comment (alex-deaconu) [reply src/environments/environment.beta.ts:2]: @ezequielc as long as setting that way doesn't break anything else, I think this is ok. Thanks.

Comment (ezequielc) [reply src/environments/environment.beta.ts:2]: @alex-deaconu Sounds good. I will leave as is for now. We can revisit in the future if we find it confusing. Thanks

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good, nice work 👍

**Pull request merged**